### DISCUSSION TITLE
Docker Image

### DISCUSSION THREAD

**Discussion by PatMis16** [jacksontj/promxy#586] [Q&A] [answered]
Hi! 
This sounds very interesting for a use case I have. 
Ist there a recent Docker Image available?
Best Regards,
Patrick

**Comment by jacksontj** [comment, marked as answer]
Always glad to see the interest! I push all the container images to quay (https://quay.io/repository/jacksontj/promxy?tab=tags) -- which is linked at the top of the README (its the `container` icon).

**Comment by jacksontj** [comment]
Oh, I should also call out that there are examples on how to deploy in a variety of systems in the repo as well: https://github.com/jacksontj/promxy/tree/master/deploy

**Reply by PatMis16** [reply]
Thank you very much! I did not see it. This is gGreat!

**Reply by PatMis16** [reply]
@jacksontj 
Hmm, the Button for the container says "canceled". Is this really up to date? Or do you recommend to build our own?

**Comment by jacksontj** [comment]
It is up-to-date, CI builds and pushes a new one on each release. It has a
weird status from when I moved to GitHub actions (quay shows status of
builds, which are now done on GitHub). I have been looking at how to fix
the icon but don't see a way to do so.
